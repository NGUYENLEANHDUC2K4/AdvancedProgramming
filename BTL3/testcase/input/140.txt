INSERT x number
BEGIN
INSERT y string
ASSIGN y 'hello'
ASSIGN z 42
END