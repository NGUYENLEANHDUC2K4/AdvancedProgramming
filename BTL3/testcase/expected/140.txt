Undeclared: ASSIGN z 42